pkg/crc/config/settings.go

```diff
@@ -103,6 +104,8 @@ func RegisterSettings(cfg *Config) {
 
 	cfg.AddSetting(KubeAdminPassword, "", ValidateString, SuccessfullyApplied,
 		"User defined kubeadmin password")
+	cfg.AddSetting(RegistryURI, constants.RegistryURI, ValidateString, SuccessfullyApplied,
+		fmt.Sprintf("Registry URI which contain bundle image (default: %s)", constants.RegistryURI))
```

Comment:
So you are saying to have `--bundle` flags works for local bundle file and also for registry? As of now `--bundle` flag is only support the bundle path locally available and we should keep that support also in case someone doesn't have local registry so they can get the bundle from image tar (using skopeo) and then use it with this option.